3.Tests/UI/StepDefinitions/User/LogoutSteps.cs

```diff
@@ -0,0 +1,42 @@
+﻿using SeleniumTest.Core.Drivers;
+using SeleniumTest.Tests.Steps.Commons;
+using TechTalk.SpecFlow;
+using Views.WebAppPages;
+
+namespace SeleniumTest.Tests.Steps.User;
+
+[Binding]
+[Scope(Feature = "User Logout")]
+public class LogoutStepDefinitions : CommonSteps
+{
+    private readonly HomePage _homePage;
+    private readonly ScenarioContext _scenarioContext;
+
+    public LogoutStepDefinitions(ScenarioContext scenarioContext) : base(scenarioContext)
+    {
+        _scenarioContext = scenarioContext;
+        _homePage = new HomePage();
+    }
+
+    [AfterScenario]
+    public void TearDown()
+    {
+        GenericWebDriver.Dispose();
```

Comment:
Move this from the steps class to a hooks class, we shouldn't use NUnit here

Reply:
We don't have a hook class yet and the `TearDown` method is not a NUnit method, its just the name that the after scenario method has thats using the Specflow package

Reply:
You're right, my bad, I confused it with a TearDown method from NUnit.